config/environments/development.rb

```diff
@@ -28,7 +28,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :amazon_dev
```

Comment:
in general, we use local storage for development and external services for staging/production environments

Reply:
+1